draft-ietf-quic-transport.md

```diff
@@ -3013,6 +3013,8 @@ guidance offered below seeks to strike this balance.
 
 An ACK frame SHOULD be generated for at least every second ack-eliciting packet.
 This recommendation is in keeping with standard practice for TCP {{?RFC5681}}.
+Every ack-eliciting packet MUST be acknowledged in at least one transmitted
+ACK frame, and SHOULD typically be acknoweldged in more than one ACK frame.
```

Comment:
I think that we want to say that every ack-eliciting packet MUST be acknowledged at least once, and ideally all packets are acknowledged until one of those acknowledgements are also acknowledged.

```suggestion
ACK frame, and SHOULD typically be acknowledged in more than one ACK frame.
```